list_members.go

```diff
@@ -64,3 +64,50 @@ func listMembers(ctx context.Context, c *client, listid string, opt ...*ListMemb
 
 	return &lmr, nil
 }
+
+func listSpecifiedUserMemberOf(ctx context.Context, c *client, userid string, opt ...*ListSpecifiedUserMemberOfOption) (*ListSpecifiedUserMemberOfResponse, error) {
+	if userid == "" {
+		return nil, errors.New("lists a specified user is a member of: id parameter is required")
+	}
+
+	lm := fmt.Sprintf(listSpecifiedUserMemberOfURL, userid)
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, lm, nil)
+	if err != nil {
+		return nil, fmt.Errorf("lists a specified user is a member of: %w", err)
+	}
+
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.bearerToken))
+
+	var lopt ListSpecifiedUserMemberOfOption
+	switch len(opt) {
+	case 0:
+		// do nothing
+	case 1:
+		lopt = *opt[0]
+	default:
+		return nil, errors.New("lists a specified user is a member of: only one option is allowed")
+	}
```

Comment:
Please validate of max_result param.

> The maximum number of results to be returned per page. This can be a number between 1 and 100. By default, each page will return 100 results.

FYI
https://developer.twitter.com/en/docs/twitter-api/lists/list-members/api-reference/get-users-id-list_memberships

Reply:
taken care of it.
https://github.com/sivchari/gotwtr/pull/97/files#diff-4a2b21004a3f8fa627bb85f26381d49ffa88e4701cb47dafefaf3974d7ec46f1R91-R101